parasol/solr/client.py

```diff
@@ -24,14 +25,38 @@
 
 logger = logging.getLogger(__name__)
 
+## NOTE: As a rule, Solr parameters that are camelcased are retained that way
+# despite not being hugely Pythonic, for consistency with Solr's responses
+# and API documentation.
 
 class QueryReponse:
     '''Thin wrapper to give access to Solr select responses.'''
     def __init__(self, response):
```

Comment:
Should we store the response itself? Is there anything else we might need from it that you're not already grabbing?